CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# v0.2.0
+Upgraded all elastic components to 7.4.0
```

Comment:
Are there any upstream changelogs we should link to?

Reply:
Added link to Upstream release notes